platform/javascript/os_javascript.cpp

```diff
@@ -253,6 +253,7 @@ static Ref<InputEventKey> setup_key_event(const EmscriptenKeyboardEvent *emscrip
 	ev->set_echo(emscripten_event->repeat);
 	dom2godot_mod(emscripten_event, ev);
 	ev->set_scancode(dom_code2godot_scancode(emscripten_event->code, emscripten_event->key));
+	ev->set_physical_scancode(dom_code2godot_scancode(emscripten_event->code, emscripten_event->key));
```

Comment:
There is actually "proper" (quoted because we are still talking about the web :( ) implementation for this. Let me do the backport on top of these changes so you could simply add it on top.

Reply:
See https://github.com/Faless/godot/commit/516eacef08ea13686126920a1a27ca484c704da6

Reply:
Thanks. I have pulled JS commit to the PR branch.